app/forms/change_note_form.rb

```diff
@@ -0,0 +1,39 @@
+class ChangeNoteForm
```

Comment:
If we were doing this for real, I assume we'd want to namespace these form objects a bit?

e.g. `app/forms/edition/change_note_form.rb` as `Edition::ChangeNoteForm`?

Reply:
Yea agreed. I'm not sure if we need `Admin::` thrown in there as well? Probs overkill since the rendering is going

Reply:
Btw i also think we should do this for other things.

We have about 6 controllers with `Edition` prepended to them. Feels like they should be name spaced too.